### ISSUE TITLE
DLP Link 3D Glasses

### ISSUE THREAD

**Issue by lilfighterr** [Microsoft/RoomAliveToolkit#76]
Hi,

How do I enable 3D such that stereo glasses will work? I notice on the projector camera settings there are values for "Stereo Separation" and "Stereo Convergence". Changing those values do not change anything. I also noticed once I run the program, the projector camera becomes disabled. 

Thanks.

**Comment by thundercarrot** [comment]
I don't believe the stereo support is working... this may be a vestiage of some earlier work. Benko may have some ideas.
